ray-operator/config/samples/ray-job.interactive-mode.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  name: rayjob-interactive-mode
+  namespace: default
+spec:
+  # InteractiveMode is only used in kubectl plugin
+  submissionMode: InteractiveMode
```

Comment:
```
jobId: `example-submission-id` 
```

And maybe add a comment explaining that a job using this submission ID needs to be created externally